tests/ui/invalid_null_ptr_usage.fixed

```diff
@@ -1,44 +1,44 @@
 fn main() {
     unsafe {
-        let _slice: &[usize] = std::slice::from_raw_parts(core::ptr::NonNull::dangling().as_ptr(), 0);
```

Comment:
Could you add a second test file with `#[no_std]` to ensure that the core suggestion still works? It can just be a copy of this file :D

Reply:
Done